.github/CODEOWNERS

```diff
@@ -36,6 +36,7 @@ rush.json       @iTwin/itwinjs-core-admins
 
 /common/**/*-extension/                       @aruniverse @calebmshafer
 
+/common/**/ecsql-*/                           @iTwin/itwinjs-core-admins @khanaffan @ColinKerr
```

Comment:
you will want to add an entry around LN 90 for owning the actual source code
`/core/ecsql/*`